static/app/components/events/interfaces/debugMeta-v2/index.tsx

```diff
@@ -518,8 +530,16 @@ class DebugMeta extends React.PureComponent<Props, State> {
 
     const displayFilter = (Object.values(filterOptions ?? {})[0] ?? []).length > 1;
 
+    const actions = (
+      <div>
+        <ToggleButton onClick={this.toggleImagesLoaded} priority="link">
+          {isOpen ? t('Hide Details') : t('Show Details')}
+        </ToggleButton>
+      </div>
```

Comment:
```suggestion
      <ToggleButton onClick={this.toggleImagesLoaded} priority="link">
        {isOpen ? t('Hide Details') : t('Show Details')}
      </ToggleButton>
```

I think these plain wrapper `div`s should be avoided if possible